tests/e2e/k8s/snapshotter-kubeconf.yaml

```diff
@@ -210,3 +230,36 @@ data:
     [image]
     public_key_file = ""
     validate_signature = false
+
+  nydusd-config.json: |-
```

Comment:
Ditto, maybe we can reuse the `nydusd-config-xxx.json`.

Reply:
Fixed.

Reply:
@imeoer  To run the kubeauth e2e test, add a custom config that changes  `enable_kubeconfig_keychain` and `enable_cri_keychain` values.